libs/coin-modules/coin-sui/src/network/sdk.test.ts

```diff
@@ -185,7 +185,7 @@ const mockTransaction = {
   ],
   timestampMs: "1742294454878",
   checkpoint: "313024",
-};
+} as SuiTransactionBlockResponse;
```

Comment:
Always odd to me using this type os assertions, telling the [compiler](https://www.totaltypescript.com/concepts/type-assertions#:~:text=Type%20assertions%20let%20you%20lie%20to%20TypeScript%20about%20what%20type%20a%20value%20is) something that might not be true. 
But i have seen it on other parts of the code base, but think we should avoid this approach overall.